stwcs/updatewcs/astrometry_utils.py

```diff
@@ -150,11 +151,13 @@ def __init__(self, url=None, raise_errors=None, perform_step=True,
         # otherwise, it will turn off raising Exceptions
         #
         self.raise_errors = False
-        if raise_errors is not None:
-            self.raise_errors = raise_errors
-            logger.info("Setting `raise_errors` to {}".format(raise_errors))
         if pipeline_error_envvar in os.environ:
             self.raise_errors = True
+        if raise_errors:
+            self.raise_errors = raise_errors
+            logger.info("Setting `raise_errors` to {}".format(raise_errors))
+        else:
+            self.raise_errors = False
```

Comment:
So the code is moved here in order to overwrite the env variable as the docs say?
I prefer the original code snippet because it's clear there's an optional value of `None` and the logger records all settings (True and False).